configs/components/razor-server.rb

```diff
@@ -0,0 +1,93 @@
+component "razor-server" do |pkg, settings, platform|
```

Comment:
Should we try to order this with the documentation format that @ScottGarman  proposed?